.release_version

```diff
@@ -1 +1 @@
-2.5.4
+2.5.3
```

Comment:
You can't roll back version numbers, the release has already been created. It's OK to increment because it is a valid change, even though it's a rollback.

Reply:
Ok, so I'll set up 2.5.5 as 2.5.4 already exists.